Kickstarter-iOS/Views/Controllers/CancelPledgeViewController.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class CancelPledgeViewController: UIViewController {
+  private let viewModel: CancelPledgeViewModelType = CancelPledgeViewModel()
+
+  // MARK: - Lifecycle
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    _ = self
+      |> \.title %~ { _ in Strings.Cancel_pledge()
+      }
```

Comment:
I know this is a styling issue but could we possibly put this on the previous line...It looks like another limitation of `SwiftFormat` 😅